pyramid/file_test.go

```diff
@@ -51,11 +51,13 @@ func TestPyramidWriteFile(t *testing.T) {
 
 	require.NoError(t, sut.Close())
 	require.NoError(t, sut.Store(ctx, filename))
+	require.NotNil(t, sut.cancelStore)
```

Comment:
I don't understand what this tests.  AFAICS it only tests _that_ a cancel function was returned, not that it has anything to do with actually cancelling the context (e.g. correctly wired up).

Reply:
Right, fixed.